sequencer.c

```diff
@@ -1497,6 +1511,17 @@ static int try_to_commit(struct repository *r,
 		setenv("GIT_COMMITTER_DATE", date, 1);
 		free(date);
 	}
+	if (opts->ignore_date) {
+		struct strbuf new_author = STRBUF_INIT;
+
+		while (*author != '>')
+			strbuf_addch(&new_author, *author++);
+
+		strbuf_addstr(&new_author, "> ");
+		datestamp(&new_author);
+
+		author = strbuf_detach(&new_author, NULL);
+	}
```

Comment:
This is the same code as above, should we factor this out into a separate function?

Reply:
...and perhaps also move the opts->committer_date_is_author_date block into that new function since it is also repeated?